tools/ci_build/github/azure-pipelines/templates/win-web-ci.yml

```diff
@@ -95,6 +95,18 @@ jobs:
       targetFolder: $(Build.SourcesDirectory)\js\web\lib\wasm\binding
       flattenFolders: true
     displayName: 'Binplace js files'
+  - script: |
+      npm i -g puppeteer
```

Comment:
Do we need to force reinstall, just in case puppeteer is installed on the target machine but not Chrom?

Reply:
checked around and looks like there is no force flag for "npm install".

The Chrome is installed via postinstall script defined in package.json (https://github.com/puppeteer/puppeteer/blob/main/packages/puppeteer/package.json#L40) so unless the npm install script failed with non-zero exitcode, I assume the chrome should be installed as expected